src/dsp_tools/models/listnode.py

```diff
@@ -150,40 +150,44 @@ def __init__(
         DELETE:
             * Not yet implemented
 
-        :param con: A valid Connection instance with a user logged in that has the appropriate permissions
-        :param id: IRI of the project [readonly, cannot be modified after creation of instance]
-        :param project: IRI of project. Only used for the creation of a new list (root node) [write].
-        :param label: A LangString instance with language dependent labels. Setting this attributes overwites all entries with the new ones. In order to add/remove a specific entry, use "addLabel" or "rmLabel". At least one label is required [read/write].
-        :param comments: A LangString instance with language dependent comments. Setting this attributes overwites all entries with the new ones.In order to add/remove a specific entry, use "addComment" or "rmComment".
-        :param name: A unique name for the ListNode (unique regarding the whole list) [read/write].
-        :param parent: Is required and allowed only for the CREATE operation. Otherwise use the "children" attribute [write].
-        :param isRootNode: Is True if the ListNode is a root node of a list Cannot be set [read].
-        :param children: Contains a list of children nodes. This attribute is only available for nodes that have been read by the method "getAllNodes()"! [read]
-        :param rootNodeIri: IRI of the root node. This attribute is only available for nodes that have been read by the method "getAllNodes()"! [read]
+        Setting a label overwites all entries. To add/remove a specific entry, use "addLabel" or "rmLabel".
+        Setting a comment overwites all entries. To add/remove a specific entry, use "addComment" or "rmComment".
+
+        Args:
+            con: A valid Connection instance with a user logged in that has the appropriate permissions
+            iri: IRI of the project [readonly, cannot be modified after creation of instance]
+            project: IRI of project. Only used for the creation of a new list (root node) [write].
+            label: LangString with lang dependent labels. At least one label is required [read/write].
+            comments:  A LangString instance with language dependent comments.
+            name: A unique name for the ListNode (unique regarding the whole list) [read/write].
+            parent: Required and allowed only for the CREATE operation. Otherwise use the "children" attribute [write].
+            isRootNode: Is True if the ListNode is a root node of a list Cannot be set [read].
+            children: list of children nodes. Only available for nodes that have been read by getAllNodes()
+            rootNodeIri: IRI of the root node. Only available for nodes that have been read by the method getAllNodes()
         """
 
         super().__init__(con)
 
         self._project = project.id if isinstance(project, Project) else str(project) if project else None
-        self._id = str(id) if id else None
+        self._id = str(iri) if iri else None
```

Comment:
why is this operation needed? it gets an optional string and converts it into an optional string?